node_modules/col-whiteboards/data/whiteboardToPng.js

```diff
@@ -42,15 +42,17 @@ var init = function() {
 
     // Create a PNG stream for these elements
     generatePng(whiteboardElements, function(err, data, dimensions) {
+      // According to https://nodejs.org/api/process.html, "rather than calling process.exit() directly,
+      // the code should set the process.exitCode and allow the process to exit naturally"
       if (err) {
         console.error(err);
-        process.exit(2);
+        process.exitCode = 2;
```

Comment:
I don't think setting the `exitCode` is going to fix anything in this case. The process is being explicitly exited because there's nothing else left to do.